### ISSUE TITLE
Feature Request: Add Accordion component to storybook.

### ISSUE THREAD

**Issue by YashKandalkar** [moja-global/FLINT-UI#158]
### Is your feature request related to a problem? Please describe.

Accordion component made for the new design was added (components/Accordion/Accordion.vue) but there is no stories file for it.

### Describe the solution you'd like.

Add a `Accordion.stories.js` file in `stories/`. (Refer to other .stories.js files to know how a component is added in storybook.js)

### Describe alternatives you've considered

-

### Additional context.

Run `yarn storybook` to check if the component is rendered properly.

**Comment by heyanurag** [comment]
I'd like to give this a try :)

**Comment by chicken-biryani** [comment]
Hey, @heyanurag definitely!

If you need help... Do reach out to us on slack (#user-interface) or under this issue.

**Comment by harsh-058** [comment]
hey,I am interested to do this ,Can i try

**Comment by chicken-biryani** [comment]
Hi, @heyanurag any updates?

Note- It's okay if you feel that you don't have the bandwidth right now to work on this issue... But do keep us in the loop

**Comment by heyanurag** [comment]
Hey @chicken-biryani sorry for being quite, I had exams this week... 
I am exploring storybook currently and will make a PR in the coming week!

**Comment by chicken-biryani** [comment]
@harsh-058 would you be okay with working on something else?

**Comment by chicken-biryani** [comment]
@harsh-058 would you be okay with working on something else?

**Comment by harsh-058** [comment]
@chicken-biryani yes sure

**Comment by sohamsshah** [comment]
Hey @heyanurag , any updates on this issue?

**Comment by heyanurag** [comment]
Hi @sohamsshah , I see there is an issue for creating a new accordion using Ant Design  #183 . I think I shall wait for the new Accordion Component to create a new `.stories.js` file right?

**Comment by sohamsshah** [comment]
Yes @heyanurag . Ideally that should be the case. Will try to unblock the #183 PR as soon as possible. Thanks for being patient 🙏

**Comment by YashKandalkar** [comment]
Closing as we didn't require a separate Accordion component, `a-collapse` can directly be used.
